waspc/data/Generator/templates/Dockerfile

```diff
@@ -47,6 +47,7 @@ WORKDIR /app
 COPY --from=server-builder /app/server/node_modules ./server/node_modules
```

Comment:
Just out of curiosity, I've noticed this line in the `Dockerfile`

```
RUN apk add --no-cache python3
```

Is there a valid reason to install `python3` or is this some sort of leftover?

Reply:
We installed it because we had people doing stuff where they needed python3, and it was missing, and it is such a common dep that people need that we decided to just ship with it.